Routers/user-router.js

```diff
@@ -0,0 +1,117 @@
+const express = require('express')
+
+const userDB = require('../data/helpers/userDb')
+
+const router = express.Router();
+
+
+//custom middleware
```

Comment:
This is good. But I would like to see this middleware abstracted out to another file. (separating concerns)